frontend/src/app/user/user.component.html

```diff
@@ -1 +1,38 @@
-<p>user works!</p>
+<div class="jumbotron">
+  <div class="container-fluid">
+    <div *ngIf="userInfo; else userError" class="container">
+      <div class="row">
+        <div class="col">
+          <img
+            class="rounded-circle"
+            src="../../assets/images/user/232960.jpeg"
+            width="250"
+            height="250"
+          />
+        </div>
+        <div class="col">
+          <h1>{{ userInfo.firstName + " " + userInfo.lastName }}</h1>
+          <p>{{ userInfo.jobTitle }}</p>
+          <p><a href="#" class="btn btn-success btn-lg">Rate me</a></p>
+        </div>
+      </div>
+      <hr class="m4-4" />
+
+      <div class="row">
+        <div class="row">
+          <ng-container *ngFor="let badge of badges">
+            <div class="col">
+              <img src="{{ badge.iconId }}" width="100" height="100" />
```

Comment:
The badges that I have designed so far, had a ratio of 4:5. But I think we can change this at a later stage.